src/Humanizer/FrequencyHumanizeExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Humanizer
+{
+    public static class FrequencyHumanizeExtensions
+    {
+        public static string Humanize(this Frequency input)
+        {
+            string output = "every";
+            switch (input.OccurrenceFrequency)
+            {
+                case FrequencyEnum.Never:
+                    output = "never";
```

Comment:
I don't really understand what you mean by translatable? Can you explain
please?
Le 29 mars 2015 23:33, "Alexander Zaytsev" <notifications@github.com> a
écrit :

> In src/Humanizer/FrequencyHumanizeExtensions.cs
> <https://github.com/MehdiK/Humanizer/pull/400#discussion_r27359388>:
>
> > +﻿using System;
> > +using System.Collections.Generic;
> > +using System.Linq;
> > +using System.Text;
> > +
> > +namespace Humanizer
> > +{
> > +    public static class FrequencyHumanizeExtensions
> > +    {
> > +        public static string Humanize(this Frequency input)
> > +        {
> > +            string output = "every";
> > +            switch (input.OccurrenceFrequency)
> > +            {
> > +                case FrequencyEnum.Never:
> > +                    output = "never";
>
> These strings need to be translatable
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MehdiK/Humanizer/pull/400/files#r27359388>.
>